keps/sig-storage/2485-read-write-once-pod-pv-access-mode/README.md

```diff
@@ -0,0 +1,950 @@
+[kubernetes/enhancements]: https://git.k8s.io/enhancements
+[kubernetes/kubernetes]: https://git.k8s.io/kubernetes
+[kubernetes/website]: https://git.k8s.io/website
+
+## Summary
+
+<!--
+This section is incredibly important for producing high-quality, user-focused
+documentation such as release notes or a development roadmap. It should be
+possible to collect this information before implementation begins, in order to
+avoid requiring implementors to split their attention between writing release
+notes and implementing the feature itself. KEP editors and SIG Docs
+should help to ensure that the tone and content of the `Summary` section is
+useful for a wide audience.
+
+A good summary is probably at least a paragraph in length.
+
+Both in this section and below, follow the guidelines of the [documentation
+style guide]. In particular, wrap lines to a reasonable length, to make it
+easier for reviewers to cite specific portions, and to minimize diff churn on
+updates.
+
+[documentation style guide]: https://github.com/kubernetes/community/blob/master/contributors/guide/style-guide.md
+-->
+
+This KEP introduces a new ReadWriteOncePod access mode for PersistentVolumes
+that restricts access to a single pod on a single node. This access mode
+differs from the existing ReadWriteOnce (RWO) access mode, which restricts
+access to a single node, but allows simultaneous access from many pods on that
+node.
+
+Additionally, this KEP outlines required changes to the CSI spec, drivers, and
+sidecars in order to support this new access mode while maintaining backwards
+compatibility.
+
+## Glossary
+
+- [Node]
+  - A virtual or physical machine in a Kubernetes cluster that runs pods
+- [PersistentVolume]
+  - A piece of storage in the cluster that has been provisioned by an
+    administrator or dynamically provisioned using `StorageClasses`
+- Access mode
+  - A description of how a PersistentVolume can be accessed
+- ReadWriteOnce (RWO)
+  - An access mode that restricts PersistentVolume access to a single node
+- ReadWriteOncePod (RWOP)
+  - A new access mode that restricts PersistentVolume access to a single pod on
+    a single node
+- [CSI]
+  - The Container Storage Interface, a specification for storage provider
+    plugins to integrate with cluster orchestrators (like Kubernetes)
+
+[Node]: https://kubernetes.io/docs/concepts/architecture/nodes/
+[PersistentVolume]: https://kubernetes.io/docs/concepts/storage/persistent-volumes/
+[CSI]: https://github.com/container-storage-interface/spec
+
+## Motivation
+
+<!--
+This section is for explicitly listing the motivation, goals, and non-goals of
+this KEP.  Describe why the change is important and the benefits to users. The
+motivation section can optionally provide links to [experience reports] to
+demonstrate the interest in a KEP within the wider Kubernetes community.
+
+[experience reports]: https://github.com/golang/go/wiki/ExperienceReports
+-->
+
+### Kubernetes Changes
```

Comment:
some related old issues you can link to:
https://github.com/kubernetes/kubernetes/issues/30085
https://github.com/kubernetes/kubernetes/issues/26567

Reply:
Done